SpaceConvertion.java

```diff
@@ -0,0 +1,33 @@
+import java.util.*;
+
+class SpaceConvertion{
+	
+	public static void main(String args[])
+	{
+		
+		String input = "This is      the program to     convert multiple spaces string into single    space string";
+		String output = "";
```

Comment:
Do we really need to store the output in a string?

Reply:
> Do we really need to store the output in a string?

We can do without the output string but if we use it would be easy to understand

Reply:
Considering space complexity, can you try removing this storing and directly print?